### PR TITLE
CLD-85: HugePages Configuration changes commit

### PR DESCRIPTION
**Pull request by sumanthravipati**

### CODE REVIEW

**Review by vitalykorolev — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -166,6 +166,14 @@ If you want to limit memory to 64GB but allow MarkLogic Server to use swap space
 ```
 To allow MarkLogic Server to use unlimited memory and swap space (up to the amount available on the host system), do not specify either "--memory" or "--memory-swap".
 
+## Configuring HugePages
+
```
Comment (jmckean17) [inline README.md:170]: Sumanth, shouldn't this section and the two before it be under "Configuration" not "Using This Image"? I suggest moving them under "Configuration".

File: README.md [hunk 2]
```diff
@@ -166,6 +166,14 @@ If you want to limit memory to 64GB but allow MarkLogic Server to use swap space
...
+By default, if HugePages are configured on the host, MarkLogic running in a container will attempt to allocate up to 3/8 of the container memory limit as HugePages. For example, consider a host with 128GB of RAM, 48GB HugePages, and running two MarkLogic containers, each with 64GB memory limit. MarkLogic in each container will only allocate up to 24GB in HugePages (3/8 * 64GB).
```
Comment (jmckean17) [inline README.md:171]: By default, if HugePages are configured on the host, the MarkLogic instance running in a container will attempt to allocate up to 3/8 of the container memory limit as HugePages. For example, consider a host with 128GB of RAM, 48GB HugePages, and running two MarkLogic containers, each with 64GB memory limit. The MarkLogic instance in each container will only allocate up to 24GB in HugePages (3/8 * 64GB).

(the MarkLogic instance)

File: README.md [hunk 3]
```diff
@@ -166,6 +166,14 @@ If you want to limit memory to 64GB but allow MarkLogic Server to use swap space
...
+
+To change this behavior, the number of HugePages MarkLogic can use can be configured on each MarkLogic container by setting the “ML_HUGEPAGES_TOTAL” environment variable to desired number of HugePages for each MarkLogic container. For example, to disable the huge pages for specific container, you would specify the following with your docker run command:
```
Comment (jmckean17) [inline README.md:173]: You can change the number of HugePages available to each MarkLogic container by setting the `ML_HUGEPAGES_TOTAL` environment variable. Set the variable for each MarkLogic container to the desired number of HugePages. For example, to disable the HugePages for specific container, specify the following with your Docker `run` command:

**Review by jmckean17 — APPROVED** [review]
Sumanth, I did a bit of rewording and a suggestion to move this and a couple of other sections to the Configuration section. Thanks!~

File: README.md (hunk 1 above)
Comment (sumanthravipati) [reply README.md:190]: Moved it. Good point

File: README.md (hunk 2 above)
Comment (sumanthravipati) [reply README.md]: Updated

File: README.md (hunk 3 above)
Comment (sumanthravipati) [reply README.md]: Updated.

**Pull request merged**